templates/area.bt

```diff
@@ -253,7 +262,7 @@ float unk3;
 float unk4;
 int unk5;
 int unk6;
-int unk7;
+int CurrentEventEntityID;
```

Comment:
What exactly is this? The name "entity id" suggests one of the typical Mabi ids, but then I would expect it to be a long.